### PR TITLE
Add matthew-carroll to reviewers

### PR DESCRIPTION
**Pull request by matthew-carroll**
Add matthew-carroll to reviewers

### CODE REVIEW

**Comment by amirh** [comment]
FYI @collinjackson Matt has volunteered to help with maintenance and reviews for these plugins.

File: CODEOWNERS [hunk 1]
```diff
@@ -6,9 +6,13 @@
 
 packages/android_alarm_manager/* @bkonyi
 packages/android_intent/* @mklim
+packages/android_intent/* @matthew-carroll
 packages/battery/* @amirh
+packages/battery/* @matthew-carroll
 packages/camera/* @bparrishMines @mklim
 packages/connectivity/* @cyanglaz
+packages/connectivity/* @matthew-carroll
```
Comment (collinjackson) [inline CODEOWNERS]: It seems more common (and consistent with this file) to put all the CODEOWNERs on one line. See this example:

https://github.com/python/cpython/blob/master/.github/CODEOWNERS

Comment (matthew-carroll) [reply CODEOWNERS]: Ok, I placed names on single lines and I also added consistent spacing to more easily read the names like the file you linked.

**Comment by collinjackson** [comment]
LGTM! Thanks.

**Comment by matthew-carroll** [comment]
@amirh @collinjackson should this PR wait to go green to merge? Or we can go ahead and merge given the non-functional changes?

**Comment by collinjackson** [comment]
Merged, thanks!

**Pull request merged**